prime-router/docs/proposals/0009-internal-tools-migration-to-superset.md

```diff
@@ -0,0 +1,50 @@
+## Background
+
+The PRIME ReportStream uses Metabase for business analytics. Metabase is a business intelligence tool that allows for the exploration, investigation, and visualization of data. With this internal tool, our team is able to generate charts and dashboards from our stored data. Even though this instrument provides quality charts, it also has its shortcomings:
+
+### Limitations 
+
+- Query complexity 
+    - Limited to simple queries only
+    
+- Data Source 
+    - Only works well with a single SQL data source
+    - Does not allow for joining of data
```

Comment:
Not sure what this means.   Many of the queries I have running in Metabase have SQL joins.   Joins can be done both by writing raw SQL, and by using their canned query generators.

Reply:
I think this particular one is about having the ability to join/correlate across data-sources (e.g. multiple databases across multiple systems), not through a `join` but logically inside superset.